### ISSUE TITLE
TF_DETERMINISTIC_OPS

### ISSUE THREAD

**Issue by StuartIanNaylor** [breizhn/DTLN#54]
run_training.py
```
# use the GPU with idx 0
os.environ["CUDA_VISIBLE_DEVICES"]='0'
# activate this for some reproducibility
os.environ['TF_DETERMINISTIC_OPS'] = '1'
```
TF_DETERMINISTIC_OPS means the model should be reproducible with the same loss, accuracy... It does slow down the training though and just wondering is there any reason why I can not just remark that out if I want as my question is that it has no other effects?

**Comment by breizhn** [comment]
Yes exactly, it just makes it more reproducible. Just comment it out and it will run faster, especially in recent TF versions.

**Comment by StuartIanNaylor** [comment]
Thanks nils I thought so but wasn't sure it has any other effect
